lib/modules/dosomething/dosomething_reportback/dosomething_reportback.cron.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @file
+ * Provides cron job/drush command for dosomething_reportback.module.
+ */
+
+/**
+ * Implements hook_cron().
+ */
+function dosomething_reportback_cron() {
+  // Get all reportbacks updated in the past hour.
+  $results = db_query("SELECT nid, sum(quantity) as quantity
+                     FROM dosomething_reportback
+                     WHERE from_unixtime(updated) >= date_add(now(), INTERVAL '-1' hour)
```

Comment:
curious why you think that?
I would think that it would be better to calculate the date in SQL, so that you know the `now()` is from that db server and not the webserver as those things might not be in sync.